kernel/aster-nix/src/process/process_vm/heap.rs

```diff
@@ -40,17 +36,12 @@ impl Heap {
 
     /// Inits and maps the heap Vmo
     pub(super) fn alloc_and_map_vmo(&self, root_vmar: &Vmar<Full>) -> Result<()> {
-        let heap_vmo = {
-            let vmo_options = VmoOptions::<Rights>::new(0).flags(VmoFlags::RESIZABLE);
-            vmo_options.alloc()?
-        };
         let vmar_map_options = {
             let perms = VmPerms::READ | VmPerms::WRITE;
             root_vmar
-                .new_map(heap_vmo, perms)
+                .new_map(PAGE_SIZE, perms)
```

Comment:
Why PAGE_SIZE is used here? Is it the minimal heap size used by Linux? Can a size 0 be used here?